package.json

```diff
@@ -1,11 +1,15 @@
 {
-  "main": "node_modules/expo/AppEntry.js",
+  "main": "src/App.tsx",
   "private": true,
   "scripts": {
     "prettier": "prettier --list-different **/*.js",
     "prettier-fix": "prettier --list-different --write **/*.js",
     "lint": "npm run prettier",
-    "test": "npm run lint"
+    "test": "npm run lint",
+    "clean": "rimraf out",
+    "build": "npm run clean && tsc",
+    "buildAndWatch": "npm run build -- -w",
+    "start": "npm run build && concurrently \"npm run buildAndWatch\" \"npm run start\""
```

Comment:
Wouldn't `npm run start` inside the `start` script lead to an infinite loop?

Reply:
Yup, fixed that 👍